### DISCUSSION TITLE
ERROR

### DISCUSSION THREAD

**Discussion by ghost** [community/community#70570] [Repositories]
### Select Topic Area

Question

### Body

File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL as well as any file permissions.
For root URLs (like http://example.com/) you must provide an index.html file.

Can someone help me? This appears when I try to host my website through github
https://github.com/ZENsalaberry/Portfolio-Personalizado

**Comment by suryaprakash251201** [comment]
i am trying but unable solve because file is missing

**Reply by ghost** [reply]
would it be?
